File: Kaiq/TwoSum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        res=[]
        count = len(nums)
        for i in range(0,count):
            a = target - nums[i];
            if a in nums[i+1:]:
                return [i,nums.index(a,i+1)] #index(value,start,end)

File: Kaiq/TwoSum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        
        vector<int> index;
        int len = nums.size();
        for(int i=0;i<len;i++){
            for(int j=i+1;j<len;j++){
                if(nums[i]==target-nums[j]){
                    index.push_back(i);
                    index.push_back(j);
                    return index;
                }
            }            
        }
    return index;          
    }
};
